src/examples/pwm/pwm.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"machine"
+	"runtime"
+)
+
+// This example assumes that an RGB LED is connected to pins 3, 5 and 6 on an Arduino.
+// Change the values below to use different pins.
+const (
+	redPin   = 3
+	greenPin = 5
+	bluePin  = 6
+)
+
+// cycleColor is just a placeholder until math/rand or some equivalent is working.
+func cycleColor(color uint8) uint8 {
+
+	if color < 10 {
+		return color + 1
+	}
+	if color > 30 && color < 200 {
+		return color + 10
+	}
+	if color > 230 {
+		return 0
+	}
+	return 0
+}
+
+func main() {
+	machine.InitPWM()
+
+	red := machine.PWM{redPin}
+	red.Configure()
+
+	green := machine.PWM{greenPin}
+	green.Configure()
+
+	blue := machine.PWM{bluePin}
+	blue.Configure()
+
+	var rc uint8
+	var gc uint8 = 20
+	var bc uint8 = 30
+
+	for {
+		rc = cycleColor(rc)
+		gc = cycleColor(gc)
+		bc = cycleColor(bc)
+
+		red.Set(rc)
+		green.Set(gc)
+		blue.Set(bc)
+
+		runtime.Sleep(runtime.Millisecond * 500)
```

Comment:
I just did an update after which you can directly use the "time" package: `time.Sleep(time.Millisecond * 500)`. See: 2a20c0c7f0c13c9b5c81422ea1c24ddd4eb765e7. This means the above code won't compile after a merge.

Reply:
Dnoe,